apps/pom.xml

```diff
@@ -541,14 +541,14 @@ checkJavaFormat() {
   cd apps
 
   set +e
-  mvn com.coveo:fmt-maven-plugin:check >/dev/null 2>&1
+  mvn com.spotify.fmt:fmt-maven-plugin:check >/dev/null 2>&1
   checkResult=$?
   set -e
 
   if [ ${checkResult} -eq 0 ]; then
     echo 'Verified Java source code format: a-okay.'
   else
-    echo "Verified Java source code format: inconsistencies found. Run 'mvn com.coveo:fmt-maven-plugin:check' for details or 'mvn com.coveo:fmt-maven-plugin:format' to automatically apply the required formatting."
+    echo "Verified Java source code format: inconsistencies found. Run 'mvn com.spotify.fmt:fmt-maven-plugin:check' for details or 'mvn com.spotify.fmt:fmt-maven-plugin:format' to automatically apply the required formatting."
   fi
```

Comment:
Outside the scope of this PR but I would prefer it if we just ran the formatter here.  If nothing changes the commit could proceed normally.  If something changes there would be unstaged changes and the commit would fail anyway.  As it stands if you missed something prior to commit you have to wait TWICE for the formatter (once for the check, again when you have to run it and restage).